scipy/stats/_distn_infrastructure.py

```diff
@@ -100,7 +100,7 @@
     (Differential) entropy of the RV.
 """
 _doc_fit = """\
-fit(data, %(shapes)s, loc=0, scale=1)
+fit(data, %(shapes)s, loc, scale)
```

Comment:
```suggestion
fit(data, %(shapes)s)
```
Perhaps these shouldn't be mentioned at all here? None of the other `kwds` are mentioned.